spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -68,7 +68,8 @@ class Analyzer(tableUtils: TableUtils,
                count: Int = 64,
                sample: Double = 0.1,
                enableHitter: Boolean = false,
-               silenceMode: Boolean = false) {
+               silenceMode: Boolean = false,
+               analyzeForMetadata: Boolean = false) {
```

Comment:
nit: `enableTimestampCheck` may be a more generic and appropriate name

Reply:
`enableHitter` also hits actual hive data. let's use `enableTimestampCheck` to keep finer granularity control